pages/register.tsx

```diff
@@ -35,7 +38,16 @@ export default function Register() {
 
   const router = useRouter();
 
-  async function handleRegister(e: FormEvent) {
+  const { mutateAsync, isPending, error } = useMutation({
+    mutationKey: QueryKeys.REGISTER,
+    mutationFn: async (formData: typeof EMPTY_FORM_DATA) =>
```

Comment:
Tämän voi nyt muuttaa myös näin:
```suggestion
    mutationFn: async (formData: FormData) =>
```